webpack.config.js

```diff
@@ -0,0 +1,104 @@
+// By default eslint assumes packages imported are supposed to be dependencies,
```

Comment:
Is this file used for anything?  There really isn't an 'npm run start' concept for the toolkit.  Could this all be put inside of the test webpack config?